### PR TITLE
desktop support for dev server ( related to #410 )

### PR DESCRIPTION
**Pull request by nnnnathann**
After some experimentation with the project, I was able to get the dev server URL working on mac desktop with a few changes to cli.cc and main.cc

### CODE REVIEW

File: src/cli/cli.cc
Comment (chicoxyzzy) [inline src/cli/cli.cc:1]: there're a couple more on lines 5395 and 5413

**Review by chicoxyzzy — APPROVED** [review]
LGTM!

File: src/cli/cli.cc
Comment (nnnnathann) [reply src/cli/cli.cc:1]: Will update the count calls to be double dash prefixed there as well, thank you

**Comment by chicoxyzzy** [comment]
Thank you @nnnnathann!

**Comment by heapwolf** [comment]
@nnnnathann awesome! :)

**Comment by lamiazar** [comment]
@nnnnathann PR `merged`, Thank you for your contribution 😊 🙏

**Pull request merged**